src/Std/Internal/Async/Process.lean

```diff
@@ -0,0 +1,239 @@
+/-
+Copyright (c) 2025 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time
+import Std.Internal.UV.System
+import Std.Data.HashMap
+
+open Std Time
+
+namespace Std
+namespace Internal
+namespace IO
+namespace Async
```

Comment:
`Async` feels like a strange place to have these, given that there is nothing asynchronous here.